gen/gen_test.go

```diff
@@ -230,7 +230,7 @@ func TestGen_writeGoDoc(t *testing.T) {
 	swapTemplate := packageTemplate
 
 	packageTemplate = `{{{`
-	err := gen.writeGoDoc(nil, nil)
+	err := gen.writeGoDoc(nil, nil, nil)
```

Comment:
could you and the new test cases having pass by configuration?

Reply:
Sure